vscode/src/ruby.ts

```diff
@@ -39,7 +40,7 @@ export class Ruby implements RubyInterface {
     outputChannel: WorkspaceChannel,
   ) {
     this.context = context;
-    this.workingFolderPath = workingFolder.uri.fsPath;
```

Comment:
I agree it's not ideal, but I think this will go away as we migrate all version managers into the new format. If you don't mind, I'll keep it as is for now.